commands/commands_test.go

```diff
@@ -0,0 +1,66 @@
+package commands
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/giantswarm/gsctl/config"
+)
+
+// create a temporary directory
+func tempDir() string {
+	dir, _ := ioutil.TempDir("", config.ProgramName)
```

Comment:
I know it's out of scope, but: I don't think ignoring this error is a good idea. You can either pass `t  *testing.T` and call `t.Fail` or `panic`.

Reply:
panic makes sense. It's used in tests only, so a non-graceful exit would be fine.